gsl/multi_span

```diff
@@ -87,7 +87,7 @@ namespace details
     template <typename SizeType>
     struct SizeTypeTraits
     {
-        static const SizeType max_value = std::numeric_limits<SizeType>::max();
+        static const SizeType max_value = (std::numeric_limits<SizeType>::max)();
```

Comment:
This is a problem with windows headers. Best dealt with differently than transmogrifying source code.  E.g. could define NOMINMAX on build command line.